src/web/app/src/components/SearchResults.tsx

```diff
@@ -18,7 +18,7 @@ const useStyles = makeStyles((theme: Theme) =>
     },
     searchResults: {
       padding: 0,
-      width: '100%',
+      width: '81vw',
```

Comment:
@Kevan-Y I think i'll revert it, but the error banner needs to be smaller then (reason for this change). I should approach it that way. Nice catch again Kevan. Really good testing!

Reply:
Instead of having a banner, we can change it to be a similar design when you get a post not found.

Not sure if we should do that in a follow-up issue, since this PR is more specific for advance search. And this issue was there before too.

Reply:
Right, however It was easily solved - I think the fix I added works, if we need more enhancement, we can create a new issue I'm ok with that.